requirements/test.txt

```diff
@@ -64,7 +64,7 @@ mccabe==0.6.1
     # via pylint
 mock==4.0.3
     # via -r requirements/test.in
-moto==2.3.2
+moto==3.0.1
```

Comment:
Dropped `Python 3.5` support. A breaking change of replacing the `mock_s3` with `mock_s3controller` for accessing existing S3 control methods. Details in https://github.com/spulec/moto/blob/master/CHANGELOG.md#300

Reply:
No usage of updated method in this repo, so it should be safe to upgrade.